californium-core/src/main/java/org/eclipse/californium/core/network/InMemoryMessageExchangeStore.java

```diff
@@ -126,6 +133,8 @@ public InMemoryMessageExchangeStore(final NetworkConfig config, TokenGenerator t
 		this.tokenGenerator = tokenProvider;
 		this.endpointIdentityResolver = endpointResolver;
 		this.config = config;
+		this.tag = StringUtil.normalizeLoggingTag(tag);
+		LOGGER.debug("{}using default TokenProvider {}", tag, RandomTokenGenerator.class.getName());
```

Comment:
default token  provider is maybe not used here.

Reply:
Ok.